CONTRIBUTING.md

```diff
@@ -1,138 +1,165 @@
 # Contributing to Ozone
 
-:tada::+1: First off, thanks for taking the time to contribute! :tada::+1:
```

Comment:
I'm quite fond of this line

:tada::+1: First off, thanks for taking the time to contribute! :tada::+1:

 It adds some colour to an otherwise very monotonous file. Leave it in please

Reply:
I see that you've added this line a little lower, but I think it greets contributors in a really pleasant, cheerful way at the top haha

Reply:
Due to how commit diffs work (and because the file undergoes major change), the diff looks messy indeed 😆